packages/components/tests/dummy/app/templates/components/pagination.hbs

```diff
@@ -148,157 +365,295 @@
+    updating the number of items listed in the page, updating the routing/URL, updating other elements in the page).</p>
+
+  <h5 class="dummy-h5">Routing/URL updates</h5>
+  <p class="dummy-paragraph">If you want the pagination to change directly the URL of the page (eg. adding/removing
+    query parameters) you need to pass the routing parameters to the component:</p>
+  <pre>🚨 TODO </pre>
+  <p class="dummy-paragraph">Renders to:</p>
+  <pre>🚨 TODO 🚨</pre>
+
+  {{! =============================== }}
+  {{! ===== PAGINATION::COMPACT ===== }}
+  {{! =============================== }}
+
+  <h4 class="dummy-h4">Pagination::Compact</h4>
+  <p class="dummy-paragraph">By default, the basic use of the pagination provides .</p>
+
+  <p class="dummy-paragraph">The basic invocation of the "compact" pagination doesn't require any argument (apart from
+    the event/routing handlers, see below):</p>
+  {{! prettier-ignore-start }}
+  {{! template-lint-disable no-unbalanced-curlies }}
+  <CodeBlock
+    @language="markup"
+    @code="
+      <Hds::Pagination::Compact />
+    "
   />
   {{! template-lint-enable no-unbalanced-curlies }}
   {{! prettier-ignore-end }}
   <p class="dummy-paragraph">Renders to:</p>
-  <Hds::Pagination::Nav @totalPages={{10}} @type="numbered" />
+  <Hds::Pagination::Compact />
+  <p class="dummy-paragraph">In this variant only the "prev" and "next" navigation controls are displayed.</p>
 
-  <h4 class="dummy-h4">“Truncated” PaginationNav</h4>
+  <h5 class="dummy-h5">Extra arguments</h5>
+  <p class="dummy-paragraph">If necessry, it's possible to hide the labels in the controls:</p>
   {{! prettier-ignore-start }}
   {{! template-lint-disable no-unbalanced-curlies }}
   <CodeBlock
     @language="markup"
-    @code='
-      <Hds::Pagination::Nav @totalPages={{1000}} @type="truncated" />
-    '
+    @code="
+      <Hds::Pagination::Compact @showLabels=\{{false}} />
+    "
   />
   {{! template-lint-enable no-unbalanced-curlies }}
   {{! prettier-ignore-end }}
   <p class="dummy-paragraph">Renders to:</p>
-  <Hds::Pagination::Nav @totalPages={{1000}} @type="truncated" />
-</section>
+  <Hds::Pagination::Compact @showLabels={{false}} />
 
-<section>
-  <h3 class="dummy-h3" id="design-guidelines"><a href="#design-guidelines" class="dummy-link-section">§</a>
-    Design guidelines</h3>
-  {{! UNCOMMENT THIS BLOCK (once the link and/or the image are available) }}
-  {{!
-  <div class="dummy-design-guidelines">
-    <p class="dummy-paragraph">
-      <a href="[ADD THE LINK TO THE FIGMA FILE/PAGE HERE!]" target="_blank" rel="noopener noreferrer">Figma UI Kit</a>
-    </p>
-    <br />
-    <img class="dummy-figma-docs" src="/assets/images/pagination-design-usage.png" alt="" role="none" />
-  </div>
-  }}
-</section>
+  <h5 class="dummy-h5">Events handling</h5>
+  <p class="dummy-paragraph">The component exposes a callback function that can be used to respond to :</p>
```

Comment:
Missing some language? Perhaps change to below:

"respond to :"
=>
"respond to page changes:"

Reply:
Fixed